content/dstu2/diagnostic-report.md

```diff
@@ -0,0 +1,26 @@
+---
+title: DiagnosticReport | FHIR DSTU 2 API
+---
+
+# DiagnosticReport
+
+* TOC
+{:toc}
+
+## Search
+
+Search for DiagnosticReports that meet supplied query parameters:
+
+    GET /DiagnosticReport?:parameters
+
+### Parameters
+
+Name | Type | Description
+-----|------|--------------
+`patient`|`string`| The identity of a patient to list reports for. Example: `12345`
```

Comment:
Following dstu2 documentation[1], use the description "The subject of the report if a patient".  For subject:Patient, use the slightly modified "The subject (Patient) of the report"

[1] http://hl7.org/fhir/2015May/diagnosticreport.html#4.21.5